colp_build/sql/name.sql

```diff
@@ -0,0 +1,110 @@
+		                                                                                        'PLAYLOT|PLGD|PLAYGRD|PLAYGND|PLYGRND|PLYGD|PLAYGRND|PLYGRD|PLGRD|PLGD.|PLYGD|PLAYGD', 
+		                                                                                        'PLAYGROUND'
+		                                                                                        ),
+		                                                                                    '@',
+		                                                                                    'AT'
+		                                                                                    ),
+		                                                                                'HSES',
+		                                                                                'HOUSES'
+		                                                                                ),
+		                                                                            ' \(GROUP 1 ',
+		                                                                            ' (GROUP 1) '
+		                                                                            ),
+		                                                                        ' PK',
+		                                                                        ' PARK'
+		                                                                        ),
+		                                                                    'STA ',
+		                                                                    'STATION '
+		                                                                    ),
+		                                                                'LIBR ',
+		                                                                'LIBRARY '
+		                                                                ),
+		                                                            'INDUST PK|IND. PARK',
+		                                                            'INSUSTRIAL PARK'
+		                                                            ),
+		                                                        'PCT |PCT. ',
+		                                                        'PRECINCT '
+		                                                        ),
+		                                                    'N.Y.C.P.D.|NYCPD',
+		                                                    'NYPD'
+		                                                    ),
+		                                                'F.H.A..',
+		                                                'FHA '
+		                                                ),
+		                                            'GDN |GARD ',
+		                                            'GARDEN '
+		                                            ),
+	                                            'AVE\)|AVE.\)',
```

Comment:
'60 BROAD ST/33 BEAVER ST' is becoming '60 BROAD ST/33 BEAVENUE ST'
another example
'CAP PROJ=NEW HAVEN -HARTFORD R' is now 'CAP PROJ=NEW HAVENUE -HARTFORD R'
and 
'GUSTAVE HARTMAN SQUARE' is now 'GUSTAVENUE HARTMAN SQUARE'

We could just not normalize Avenue or it needs to be that there is at least a space before and after 'AVE' if 'AVE' does not end the string